lib/instrumentation/http-shared.js

```diff
@@ -130,7 +130,7 @@ exports.traceOutgoingRequest = function (agent, moduleName, method) {
     return function (...args) {
       // TODO: See if we can delay the creation of span until the `response`
       // event is fired, while still having it have the correct stack trace
-      var span = agent.startSpan(null, 'external', moduleName, 'http')
+      var span = agent.startSpan(null, 'external', 'http')
```

Comment:
nit: It looks like `span.action = req.method` could be removed by passing in req.method as the fourth argument here.

Reply:
We don't have `req` until we do `      var req = orig.apply(this, newArgs)` a little while *after* the span is created.